apps/web/app/(authenticated)/(app)/app/apis/[apiId]/settings/update-api-name.tsx

```diff
@@ -24,19 +32,31 @@ type Props = {
 };
 
 export const UpdateApiName: React.FC<Props> = ({ api }) => {
-  const { pending } = useFormStatus();
+  const form = useForm<z.infer<typeof formSchema>>({
+    resolver: zodResolver(formSchema),
+    defaultValues: {
+      name: api.name,
+      apiId: api.id,
+      workspaceId: api.workspaceId,
+    },
+  });
 
+  const updateName = trpc.api.updateName.useMutation({
+    onSuccess() {
+      toast.success("Your API name has been renamed!");
+      router.refresh();
+    },
+    onError(err) {
+      console.log(err);
+      toast.error(err.message);
+    },
+  });
+  async function onSubmit(values: z.infer<typeof formSchema>) {
+    updateName.mutateAsync(values);
+  }
+  const router = useRouter();
```

Comment:
does this work? we're importing it here, but already use it in line 47

Reply:
Thought I moved this but must have missed it on this file.